docs/glossary.md

```diff
@@ -242,6 +254,12 @@ When service to the primary Cluster is compromised, a [failover](#failover) tran
 
 _Tags: [term](/tags/term), [explanation](/tags/explanation)_
 
+#### [Handler](/nexus#handler)
```

Comment:
Hmmmmm, are we generally calling this a "handler" and not a "Nexus handler"? We use "handles" a lot in Core SDKs.